jekyll/_sass/_instantsearch.scss

```diff
@@ -46,41 +67,68 @@
 }
 
 .result-item-wrap {
-  padding: $spacing-small-vertical;
-  border-bottom: 1px solid $gray-200;
+  $result-spacing: $spacing-medium-vertical / 2;
 
-  a strong {
-    font-family: "roboto";
-    font-size: $font-size-small;
+  padding: $result-spacing/2 $result-spacing;
+  margin-left:  -$result-spacing;
+  margin-right: -$result-spacing;
+
+  .result-title {
+    font-size: 18px;
+    font-weight: $font-weight-bold;
+
+    .fa-external-link {
+      margin-left: 4px;
+      font-size: 80%;
+    }
   }
 
-  .content {
-    margin-top: 4px;
-    font-size: $font-size-small;
+  .result-content {
+    margin-top: 8px;
   }
 
-  .context {
+  .result-context {
     margin-top: 4px;
     color: $gray-dark;
-    font-size: $font-size-micro;
+    font-size: $font-size-small;
+  }
+
+  .result-tag {
+    font-size: 12px;
+    font-weight: 500;
+    text-transform: uppercase;
+    letter-spacing: 1.7px;
+    padding: 4px 8px;
+    margin-right: 12px;
+    color: #343434;
+    background-color: white;
+    border: 0.5px solid #D8D8D8;
   }
 
-  em {
-    font-style: normal;
-    font-weight: bold;
+  mark {
+    color: #1A649A;
+    background-color: #EEF2F5;
```

Comment:
The design has this as a dark blue (`#19639A`) with a 5% opacity.
Using transparency allows the contrast of the `mark` background colour compared to the result background colour to be more similar between the hover and non-hover state of the result (white vs grey background)